etc/kayobe/compute.yml

```diff
@@ -179,6 +179,15 @@
 # compute_qemu_conf_extra.
 #compute_qemu_conf:
 
+# Whether to enable libvirt SASL authentication. Default is true.
```

Comment:
Do we need a config sync somewhere?

Reply:
Think so, will check when these keys landed upstream.